counsel.el

```diff
@@ -4677,6 +4682,47 @@ as arguments."
           (const :tag "Command" counsel-linux-app-format-function-command-only)
           (function :tag "Custom")))
 
+(defface counsel-application-name
+  '((t :inherit font-lock-builtin-face))
+  "Face for displaying executable names."
+  :group 'ivy-faces)
+
+(defface counsel-org-outline-1
+  '((t :inherit org-level-1))
+  "Face for displaying org headings.")
+
+(defface counsel-org-outline-2
+  '((t :inherit org-level-2))
+  "Face for displaying org headings.")
+
+(defface counsel-org-outline-3
+  '((t :inherit org-level-3))
+  "Face for displaying org headings.")
+
+(defface counsel-org-outline-4
+  '((t :inherit org-level-4))
+  "Face for displaying org headings.")
+
+(defface counsel-org-outline-5
+  '((t :inherit org-level-5))
+  "Face for displaying org headings.")
+
+(defface counsel-org-outline-6
+  '((t :inherit org-level-6))
+  "Face for displaying org headings.")
+
+(defface counsel-org-outline-7
+  '((t :inherit org-level-7))
+  "Face for displaying org headings.")
+
+(defface counsel-org-outline-8
+  '((t :inherit org-level-8))
+  "Face for displaying org headings.")
+
+(defface counsel-default-outline
+  '((t :inherit minibuffer-prompt))
+  "Face for displaying default headings.")
```

Comment:
The name can be changed of course, but this has nothing to do with org.

Reply:
> but this has nothing to do with org.

It could, depending on user settings, but you're right that it's not **specific** to Org. In that case, my only request is that the docstring be clarified, as "default headings" aren't an established concept. How about "Default face for displaying outline headings" instead?